gensim/models/base_any2vec.py

```diff
@@ -28,18 +28,40 @@
 
 class BaseAny2VecModel(utils.SaveLoad):
     """Base class for training, using and evaluating any2vec model.
-    Contains implementation for multi-threaded training.
+
```

Comment:
Fix description in the head of current file please (make it more detailed, links to several related classes, etc).